src/me/StevenLawson/TotalFreedomMod/Command_wipewarpdata.java

```diff
@@ -0,0 +1,29 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import java.io.File;
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.SENIOR, source = SourceType.ONLY_CONSOLE)
```

Comment:
Any reason it has to be console only? If its senior only already maybe we could allow it in game? Just an idea, something for further discussion.